nevergrad/optimization/optimizerlib.py

```diff
@@ -17,11 +17,11 @@
 from nevergrad.parametrization import discretization
 from nevergrad.parametrization import _layering
 from nevergrad.parametrization import _datalayers
-from . import callbacks
 from . import oneshot
 from . import base
 from . import mutations
-from . import utils
+from .metamodel import MetaModelFailure as MetaModelFailure
+from .metamodel import _learn_on_k_best as _learn_on_k_best
```

Comment:
```suggestion
from . import metamodel
```
then use metamodel.MetaModelFailure etc...
That makes it clearer where things come from.

Reply:
Done.